src/Autonomous.h

```diff
@@ -3,6 +3,8 @@
 #include <cmath>
 #include <memory>
 #include <vector>
+#include <future>
+#include <thread>
```

Comment:
This is a minor thing, but is this import used anywhere? `async`, `future`, and `future_status` all seem to be a part of the `future` header.